<ui version="4.0" >
 <class>gascontrol</class>
 <widget class="QWidget" name="gascontrol" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../rsc/gasman.qrc" >
    <normaloff>:/images/gasman_sm.png</normaloff>:/images/gasman_sm.png</iconset>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font" >
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title" >
        <string>Simulation Controls</string>
       </property>
       <layout class="QHBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Weight</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Circuit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Speed</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>View</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QDoubleSpinBox" name="weightDoubleSpinBox" />
          </item>
          <item>
           <widget class="QComboBox" name="circuitComboBox" />
          </item>
          <item>
           <widget class="QComboBox" name="speedComboBox" />
          </item>
          <item>
           <widget class="QComboBox" name="viewComboBox" />
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="specialGroupBox" >
         <property name="title" >
          <string>Special</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QRadioButton" name="disUpRadioButton" >
            <property name="text" >
             <string>Disable Uptake</string>
            </property>
            <property name="autoExclusive" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="disRetRadioButton" >
            <property name="text" >
             <string>Disable Return</string>
            </property>
            <property name="autoExclusive" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="enVapRadioButton" >
            <property name="text" >
             <string>Enable Vapor</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="runPushButton" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text" >
          <string>Begin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="afapPushButton" >
         <property name="text" >
          <string>AFAP</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="agentGroupBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title" >
      <string>Agents</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="upPushButton" >
          <property name="maximumSize" >
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../rsc/gasman.qrc" >
            <normaloff>:/images/up.png</normaloff>:/images/up.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="downPushButton" >
          <property name="minimumSize" >
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../rsc/gasman.qrc" >
            <normaloff>:/images/down.png</normaloff>:/images/down.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="agentTableWidget" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy" >
         <enum>Qt::ActionsContextMenu</enum>
        </property>
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="verticalScrollBarPolicy" >
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="editTriggers" >
         <set>QAbstractItemView::AllEditTriggers</set>
        </property>
        <property name="alternatingRowColors" >
         <bool>true</bool>
        </property>
        <property name="showGrid" >
         <bool>true</bool>
        </property>
        <property name="gridStyle" >
         <enum>Qt::DotLine</enum>
        </property>
        <property name="cornerButtonEnabled" >
         <bool>false</bool>
        </property>
        <property name="rowCount" >
         <number>1</number>
        </property>
        <row/>
        <column>
         <property name="text" >
          <string>Agent</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Color</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rsc/gasman.qrc" />
 </resources>
 <connections/>
</ui>
